Paco-iOS/NSObject+ITNDescription.m

```diff
@@ -0,0 +1,224 @@
+//
+//  NSObject+ITNDescription.m
```

Comment:
In keeping with our current project layout, please put these somewhere under the Paco/source directory, a PacoSerializers folder, like in the Xcode project sounds about right.